platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -403,6 +403,7 @@ In style specification | Method, function, or predicate type | Format string syn
 `zoom`                 | `NSExpression.zoomLevelVariableExpression` | `$zoomLevel`
 `heatmap-density`      | `NSExpression.heatmapDensityVariableExpression` | `$heatmapDensity`
 `line-progress`        | `NSExpression.lineProgressVariableExpression` | `$lineProgress`
+`format`               | `mgl_attributed:` | `mgl_attributed({x, y, z})`
```

Comment:
Should we extend NSExpression with an initializer corresponding to this operator?

Reply:
I think makes sense.